SIT/regression/03_performance/fix.properties

```diff
@@ -0,0 +1,16 @@
+spark.cdm.origin.connect.host                     cdm-sit-cass
+spark.cdm.target.connect.host                     cdm-sit-cass
+
+spark.cdm.schema.origin.keyspaceTable             origin.regression_performance
```

Comment:
I would vote to rename these properties so that it is `spark.cdm.[origin|target].schema.<whatever>` convention so it's in-line with other properties.